xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/js/xwiki/usersandgroups/usersandgroups.js

```diff
@@ -71,6 +78,9 @@ window.MSCheckbox = Class.create({
     //add new image
     var img = document.createElement('img');
     img.src = this.images[state];
+    img.alt = this.domNode.getAttribute('data-title') + " " + this.alts[state];
+    img.tabIndex = 0;
```

Comment:
Isn't it a bad accessibility practice to add focus to elements that don't support it natively? Wouldn't it be better to wrap the image in a button?

Reply:
I wrapped the image in a button and moved most of the behavior to this button. :+1: